package.json

```diff
@@ -13,7 +13,6 @@
     "@ardrive/ardrive-promise-cache": "^1.1.4",
     "@ardrive/turbo-sdk": "^1.0.1",
     "@permaweb/aoconnect": "^0.0.56",
-    "ajv": "^8.12.0",
```

Comment:
ajv is only used by express middleware so removing here and letting that dep decide what version to load and removing here